packages/commands/src/bitcoin/depositAndCall.ts

```
@@ -0,0 +1,274 @@
+  console.log("Commit TXID:", commitTxid);
+
+  const revealHex = await makeRevealTransaction(
+    commitTxid,
+    0,
+    amountSat,
+    options.gateway,
+    10,
+    {
+      internalKey: commit.internalKey,
+      leafScript: commit.leafScript,
+      controlBlock: commit.controlBlock,
```

ESLint

[error] Expected object keys to be in ascending order. 'controlBlock' should be before 'leafScript'.

(sort-keys-fix/sort-keys-fix)